kitchen.dokken.yml

```diff
@@ -23,9 +23,9 @@ platforms:
     driver:
       image: dokken/centos-8
       pid_one_command: /usr/lib/systemd/systemd
-  - name: fedora-31
+  - name: fedora-32
     driver:
-      image: dokken/fedora-31
+      image: dokken/fedora-32
```

Comment:
Can you please this to use `fedora-latest` instead so we don't have to keep updating this? Also please update this in the other places this is mentioned.

Reply:
I've personally steered away from this due to the change rate in Fedora and it breaking CI 'without' any changes between Fedora versions. But if that's the way we'd rather go I'll change it over.